ITIS_HW_1_BULLY.playground/Contents.swift

```diff
@@ -0,0 +1,746 @@
+import UIKit
+
+protocol Fraction{
+    var name: String {get set}
+    var aliveStudents: [Student] {get set}
+    var deadStudents: [Student] {get set}
+    
+    func isDead() -> Bool
+    
+    func Attack(attackedFraction: Fraction)
+    
+    func lookForLoot()
+    
+    func updateInfoAboutStudents()
+    
+    func reviveAllStudents()
+}
+
+protocol Student{
+    var hp: Int {get set}
+    var hpMax: Int{get set}
+    var damage: Int {get set}
+    var fraction: String {get set}
+    var shoutOut: String {get set}
+    
+    func isDead()->Bool
+    
+    func AttackBully(student: Bully)
```

Comment:
```suggestion
    func attackBully(student: Bully)
```
все функции с маленькой